src/sass/components/_nested-navigation.scss

```diff
@@ -0,0 +1,127 @@
+.#{$prefix}-nested-nav {
+  &__list {
+    flex-basis: 100%;
+    list-style-type: none;
+    padding-left: 0;
+  }
+
+  &__list[hidden],
+  &__list[aria-hidden='true'] {
+    display: none;
+  }
+
+  &__item {
+    align-items: center;
+    border-top: 1px solid $color-black--100;
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 0; // resets Rivet default margin-top
+  }
+
+  &__link {
+    color: $color-black;
+    flex-grow: 1;
+    padding: $sm;
+    text-decoration: none;
+
+    &[aria-current] {
+      box-shadow: inset $xxs 0 0 $color-blue;
+    }
+
+    &:focus,
+    &:hover {
+      color: $color-blue;
+      text-decoration: underline;
+    }
+  }
+
+  &__icon {
+    align-items: center;
+    background: transparent;
+    border: none;
+    justify-content: center;
+    padding: $sm;
```

Comment:
See comment above. You will need to adjust this value to match what ever reduced padding value we end up settling on for the link.